specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/Jobs.json

```diff
@@ -820,6 +823,9 @@
       "type": "object",
       "description": "Container Apps Job resource specific properties.",
       "properties": {
+        "extendedLocation": {
```

Comment:
update examples

Reply:
Added a sample for createorupdate container app on connected environment.